inject.js

```diff
@@ -667,7 +667,7 @@ function initializeNow(document) {
     );
   });
   observer.observe(document, {
-    attributeFilter: ["aria-hidden"],
+    attributes: true,
```

Comment:
This is a heavy performance tax, instead of opting in wholesale, we can provide a list of attributes. Is there an attribute we can target for AppleTV?

Reply:
yes there is, I'm a little worried about it changing cuz it's kind of random, but yeah the perf tax might be high otherwise